js/client-mainmenu.js

```diff
@@ -1175,6 +1244,38 @@
 				teambuilder.selectFolder(teamFormat);
 			}
 		},
+		selectFolder: function (key) {
+			var sourceEl = this.sourceEl;
+			var team = this.team;
+			var format = this.format;
+			var room = this.room;
+			var folderToggleOn = this.folderToggleOn;
+			var folderExpanded = this.folderExpanded;
+			var moreTeams = this.moreTeams;
+			var keyExists = false;
+			for (var i = folderExpanded.length - 1; i > -1; i--) {
```

Comment:
This is a place that might be worth commenting on. Though [`Array#filter`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter) is probably a better bet here. It probably doesn't need a polyfill in `battle-dex.ts`.

Reply:
If it needs a polyfill, I'll add it myself later.

Reply:
@kanmingxu00